src/gui/src/scriptWidget.cpp

```diff
@@ -208,9 +225,19 @@ void ScriptWidget::addLogToOutput(const QString& text, const QColor& color)
   addToOutput(text, color);
 }
 
-void ScriptWidget::addReportToOutput(const QString& text)
+bool ScriptWidget::reportTimerIsActive()
+{
+  return report_timer_->isActive();
+}
```

Comment:
this might be a overkill since the single line of code it wraps is fairly self-explanatory. (this is a stylistic thing, so nothing major here)